tests/phpunit/tests/html-api/wpHtmlProcessorHtml5lib.php

```diff
@@ -200,6 +186,13 @@ private static function build_tree_representation( ?string $fragment_context, st
 						}
 					}
 
+					// Self-contained tags contain their inner contents as modifiable text.
+					$modifiable_text = $processor->get_modifiable_text();
+					if ( '' !== $modifiable_text ) {
+						$output .= str_repeat( $indent, $indent_level ) . "\"{$processor->get_modifiable_text()}\"\n";
```

Comment:
Did you intend to re-use `$modifiable_text` here?

```suggestion
						$output .= str_repeat( $indent, $indent_level ) . "\"{$modifiable_text}\"\n";
```

Reply:
Reused in [a7f884c](https://github.com/WordPress/wordpress-develop/pull/6972/commits/a7f884c48c763d66149ed964c895abb083af127a)